### ISSUE TITLE
Icon: invalid proptype color

### ISSUE THREAD

**Issue by RWOverdijk** [react-native-training/react-native-elements#325]
Using Icon, a simple iconStyle with a color gives me a warning, **but it still works**. The warning is just annoying :) I don't know what the correct type to use would be here, otherwise I'd submit a PR

```jsx
<Icon iconStyle={{color: 'red'}} />
```

<img width="583" alt="screenshot 2017-04-21 08 04 57" src="https://cloud.githubusercontent.com/assets/781745/25264746/3c28b81a-2669-11e7-9c5d-f5c2f74f2837.png">

**Comment by bozzmob** [comment]
Hello,

Thanks for raising the issue, this will be fixed as a part of #323 . Will update once PR is ready.

**Comment by binoy14** [comment]
Closing this in favor of #323

**Comment by bozzmob** [comment]
This has been fixed and will be available from v0.11.2

**Comment by RWOverdijk** [comment]
Happiness.
